activities/Measure.activity/locale.ini

```diff
@@ -112,7 +130,13 @@ PastMonth=El mes pasado
 PastYear=El año pasado
 UserJoined={{name}} joined
 UserLeft={{name}} left
-
+FrequencyBase=Base de frecuencia
```

Comment:
Do not try to localize in Spanish. Just let strings in English. It's better to let native language speaker do localization.